types/rest/client.d.ts

```diff
@@ -33,7 +33,7 @@ export class Client {
     hostedMessagingNumber: HostedMessagingNumberInterface;
     maskingSession:MaskingSessionInterface;
     tollfreeVerification: TollfreeVerificationInterface;
-    verifyCallerId:VerifyCallerIdInterface;
+    verifyCallerId:VerifyInterface;
```

Comment:
To avoid confusion , can we change every name to verifycaller ID itself

Reply:
Done